docs/pages/versions/v52.0.0/sdk/safe-area-context.mdx

```diff
@@ -99,7 +99,8 @@ function HookComponent() {
 
 <APIBoxSectionHeader text="Returns" />
 
-<CornerDownRightIcon className="icon-sm mr-2 inline-block align-middle text-icon-secondary" /> [`EdgeInsets`](#edgeinsets)
+<CornerDownRightIcon className="icon-sm mr-2 inline-block align-middle text-icon-secondary" />
+[`EdgeInsets`](#edgeinsets)
```

Comment:
I think the current version of it is fine (the symbol and the returned value are supposed to be in the same line):

![CleanShot 2025-03-12 at 19 42 43](https://github.com/user-attachments/assets/09604f1e-f9e3-4344-89ca-e8c2bc0a963e)

Version from PR:


![CleanShot 2025-03-12 at 19 42 15](https://github.com/user-attachments/assets/188f8e13-c00a-4d85-8325-edc5c494b86b)

Reply:
Heh, the manually constructed APIBoxes in general does not work right, but that's the problem for an another PR in the future. 😅 Will preventing these reformats with `prettier-ignore` will be viable solution for now?

Reply:
Ah I totally forgot that this was caused by manual APIBoxes usage 😅 
I think it's fine for now to ship this as it is. I'll create a task for this to follow up on the component level. Thank you!

Reply:
I've created a task for this, so marking this comment resolved.